<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro_ok</class>
 <widget class="QWidget" name="registro_ok">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Clowns_Password.ico</normaloff>../img/Clowns_Password.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="enviar_email">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="codigo_trocar_senha">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-style:outset;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="verificador">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>authenticate</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_informa">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enviar_codigo_email">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>SEND CODE</string>
   </property>
  </widget>
  <widget class="QLabel" name="enviar_email_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_enviar_codigo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-style:outset;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
